spec.bs

```diff
@@ -248,6 +251,92 @@ If using a centralized mechanism for fetching keys, user agents should have a re
 
 When using a registration process, it is recommended that user agents apply an expiration date to registration requests, to allow for the removal of deprecated or no longer active issuers.
 
+VOPRF Methods {#voprf-methods}
+====================================
+
+This document encodes protocol messages in TLS notation from [[!RFC8446]].
+
+For Issue (corresponding to **BlindEvaluate** function of the [[!VOPRF]] protocol), the Request and Response are serialized as:
```

Comment:
Make "Issue" a Bikeshed `<dfn>` so that other places in the spec can link to it. Then define an "anchors" section as described by https://speced.github.io/bikeshed/#custom-dfns with BlindEvaluate linking to the specific place in VOPRF that defines it. Then you'll use [=BlindEvaluate=] to link there.